traffic_ops/traffic_ops_golang/trafficvault/backends/postgres/url_sig_keys.go

```diff
@@ -24,6 +24,7 @@ import (
 	"database/sql"
 	"encoding/json"
 	"errors"
+	"github.com/apache/trafficcontrol/lib/go-util"
```

Comment:
this should be grouped below with the other `github.com/apache/trafficcontrol/` import